pom.xml

```diff
@@ -58,69 +34,57 @@
-        <groupId>org.eluder.coveralls</groupId>
-        <artifactId>coveralls-maven-plugin</artifactId>
-        <version>4.3.0</version>
-        <!-- Explicit dependency on jaxb-api to avoid problems with
-          JDK9 and later, until a new release is made:
-          See also https://github.com/trautonen/coveralls-maven-plugin/issues/112
-        -->
-        <dependencies>
-          <dependency>
-            <groupId>javax.xml.bind</groupId>
-            <artifactId>jaxb-api</artifactId>
-            <version>2.3.1</version>
-          </dependency>
-        </dependencies>
-      </plugin>
-      <!-- spotbugs is defined in parent plugin-pom but overriding some things here -->
-      <plugin>
-        <groupId>com.github.spotbugs</groupId>
-        <artifactId>spotbugs-maven-plugin</artifactId>
-        <executions>
-          <execution>
-            <id>spotbugs</id>
-            <goals>
-              <goal>check</goal>
-            </goals>
-            <phase>verify</phase>
-            <configuration>
-              <failOnError>${spotbugs.failOnError}</failOnError>
-              <spotbugsXmlOutput>false</spotbugsXmlOutput>
-              <effort>${spotbugs.effort}</effort>
-              <threshold>${spotbugs.threshold}</threshold>
-              <onlyAnalyze>hudson.plugins.jira.*</onlyAnalyze>
-            </configuration>
-          </execution>
-        </executions>
-      </plugin>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-surefire-plugin</artifactId>
-        <configuration>
-          <reuseForks>false</reuseForks>
-          <!-- fix issues in test when trying to serialize some mocks and to run some tests within your IDE you will need this -->
-          <systemPropertyVariables>
-            <hudson.remoting.ClassFilter>hudson.plugins.jira.JiraFolderProperty,hudson.plugins.jira.JiraSite,hudson.plugins.jira.JiraJobAction,com.atlassian.jira.rest.client.api.domain.IssueType</hudson.remoting.ClassFilter>
-          </systemPropertyVariables>
-        </configuration>
-      </plugin>
-    </plugins>
-  </build>
+  <scm>
+    <connection>scm:git:https://github.com/${gitHubRepo}.git</connection>
+    <developerConnection>scm:git:git@github.com:${gitHubRepo}.git</developerConnection>
+    <tag>${scmTag}</tag>
+    <url>https://github.com/${gitHubRepo}</url>
+  </scm>
+
+  <issueManagement>
+    <system>JIRA</system>
+    <url>https://issues.jenkins-ci.org/browse/JENKINS</url>
+  </issueManagement>
+
+  <properties>
+    <revision>3.11</revision>
+    <changelist>-SNAPSHOT</changelist>
+    <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
+    <jira-rest-client.version>5.2.5</jira-rest-client.version>
+    <fugue.version>4.7.2</fugue.version>
+    <!-- jenkins -->
+    <jenkins.version>2.375.2</jenkins.version>
+    <!-- security spotbugs -->
+    <spotbugs.failOnError>false</spotbugs.failOnError>
+    <spotless.check.skip>false</spotless.check.skip>
```

Comment:
- should that be changed to true?
- should I also see `.mvn_exec_spotless` in this PR?

Reply:
> should that be changed to true?

It should not.

> should I also see `.mvn_exec_spotless` in this PR?

You should not.

Reply:
And this is because? Is your "How do I adopt this?" section outdated from https://github.com/jenkinsci/plugin-pom/pull/733?

How about referencing this somewhere on https://www.jenkins.io/doc/developer/development-environment/ide-configuration/?

Reply:
https://github.com/jenkinsci/plugin-pom/pull/733 is outdated. For current documentation, see:

https://github.com/jenkinsci/plugin-pom#formatting